static/app/views/relocation/uploadBackup.tsx

```diff
@@ -0,0 +1,248 @@
+import {useContext, useRef, useState} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {Client} from 'sentry/api';
+import {Button} from 'sentry/components/button';
+import {RelocationOnboardingContext} from 'sentry/components/onboarding/relocationOnboardingContext';
+import Well from 'sentry/components/well';
+import {IconFile, IconUpload} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import {space} from 'sentry/styles/space';
+import testableTransition from 'sentry/utils/testableTransition';
+import useApi from 'sentry/utils/useApi';
+import StepHeading from 'sentry/views/relocation/components/stepHeading';
+
+import {StepProps} from './types';
+
+type UploadWellProps = {
+  centered: boolean;
+  draggedOver: boolean;
+  onDragEnter: Function;
+  onDragLeave: Function;
+  onDragOver: Function;
+  onDrop: Function;
+};
+
+export function UploadBackup(__props: StepProps) {
+  const api = useApi({
+    api: new Client({headers: {Accept: 'application/json; charset=utf-8'}}),
+  });
+  const [file, setFile] = useState<File>();
+  const [dragCounter, setDragCounter] = useState(0);
+  const inputFileRef = useRef<HTMLInputElement>(null);
+  const relocationOnboardingContext = useContext(RelocationOnboardingContext);
+  const user = ConfigStore.get('user');
+
+  const handleDragEnter = event => {
+    event.preventDefault();
+    setDragCounter(dragCounter + 1);
+  };
+
+  const handleDragLeave = () => {
+    setDragCounter(dragCounter - 1);
+  };
+
+  const handleDrop = event => {
+    event.preventDefault();
+    setDragCounter(0);
+
+    setFile(event.dataTransfer.files[0]);
+  };
+
+  const handleFileChange = event => {
+    const newFile = event.target.files?.[0];
+
+    // No file selected (e.g. user clicked "cancel")
+    if (!newFile) {
+      return;
+    }
+
+    setFile(newFile);
+  };
+
+  const onFileUploadLinkClick = () => {
+    inputFileRef.current && inputFileRef.current.click();
+  };
+
+  const handleStartRelocation = async () => {
+    const {orgSlugs, regionUrl} = relocationOnboardingContext.data;
+    if (!orgSlugs || !regionUrl || !file) {
+      addErrorMessage(
+        t(
+          'An error has occured while trying to start relocation job. Please contact support for further assistance.'
+        )
+      );
+      return;
+    }
+    const formData = new FormData();
+    formData.set('orgs', orgSlugs);
+    formData.set('file', file);
+    formData.set('owner', user.email);
+    try {
+      await api.requestPromise(`/relocations/`, {
+        method: 'POST',
+        host: regionUrl,
+        data: formData,
+      });
+
+      addSuccessMessage('Triggered Relocation Job');
```

Comment:
nit: let's make this copy a bit less engineer-y, and more UX focused:

```suggestion
      addSuccessMessage("Your relocation has started - we'll email you with updates as soon as we have 'em!");
```

Reply:
done